.storybook/main.js

```diff
@@ -41,6 +39,11 @@ module.exports = {
 			],
 			include: path.resolve(__dirname, "../"),
 		});
+		config.plugins.push(
```

Comment:
add back buffer support that is necessary to handle images (that is removed webpack 5)